vue/components/ui/organisms/listing/listing.vue

```diff
@@ -208,6 +239,19 @@ body.mobile .listing {
     vertical-align: middle;
 }
 
+.listing .container-ripe .search-container {

```

Comment:
Won't be needed with the slot.

Reply:
I still need the div for the `mouseover` and `mouseleave` capture

Reply:
The tooltip should take care of that already. I don't know why but your tooltip seems different from this one: https://github.com/ripe-tech/ripe-white/blob/master/vue/components/molecules/tooltip/tooltip.vue